apps/pm-user/src/components/molecules/quickInfoBar/QuickInfoBar.js

```diff
@@ -0,0 +1,36 @@
+import React from 'react';
+import { Segmented } from '../../atoms';
+import { SEGMENT_OPTIONS } from './constants';
+import { useHistory } from 'react-router-dom';
+import styles from './quickInfoBar.module.scss';
+
+const QuickInfoBar = () => {
+  const history = useHistory();
+
+  //redirecting on changing the value
+  function handleRedirectOnClick(value) {
+    // switch (value) {
+    //     case "received" :
+    //         return history.pushs("/recieved");
+    //     case "sent" :
+    //         return history.pushs("/sent");
+    //     case "accepted" :
+    //         return history.pushs("/accepted");
+    //     default :
+    //         return history.pushs("/recieved");
+    // }
+  }
+
+  return (
+    <div className={styles.quickInfoBar}>
+    <Segmented
+      block={true}
+      defaultValue="received"
+      onChange={handleRedirectOnClick}
+      options={SEGMENT_OPTIONS}
+    />
+    </div>
+  );
+};
+
+export default QuickInfoBar;
```

Comment:
Why are you not taking props in QuickInfoBar component ?
Our end goal is to make this component reusable... right?
Then we must take props.

Reply:
No it only one time use thing and it not having any props here.

Reply:
But for different use we have to show different data.... So, sideBar component should take props....

Reply:
either sideBar or QuickInfoBar should take props...